cognite_toolkit/cognite_modules/examples/cdf_data_pipeline_3d_valhall/functions/functions.yaml

```diff
@@ -10,8 +10,6 @@
     # The below two environment variables are set by the Toolkit
     ENV_TYPE: '${CDF_BUILD_TYPE}'
     CDF_ENV: '${CDF_ENVIRON}'
-  # Number of cores, not available in Azure
-  cpu: 0.60
```

Comment:
Instead let the server set the default based on whether it is deployed on Azuer/AWS/Google